util/src/main/java/io/kubernetes/client/Exec.java

```diff
@@ -47,6 +54,7 @@
   private static final Logger log = LoggerFactory.getLogger(Exec.class);
 
   private ApiClient apiClient;
+  private long graceTimeoutMs = 5_000L;
```

Comment:
Given that this is only used in one method, make this a parameter instead? (or delete)

Reply:
I'll delete it then.